public/kibi_timeline.js

```diff
@@ -238,6 +230,12 @@ define(function (require) {
                   }
                 }
                 events.push(e);
+              } else {
+                if ($scope.notifyDataErrors || $scope.notifyDataErrors === '') {
```

Comment:
Instead of checking here if it is  === ''
It would be better to set the default value to false here 
https://github.com/sirensolutions/kibi_timeline_vis/blob/master/public/kibi_timeline_vis.js#L22